models/topicsModels.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../db/connection.js')
+
+exports.getAllTopics =()=>{
+
+    return db
+    .query(`SELECT * FROM topics; `)
+    .then(topics => {
+        return topics.rows;
+      })
+      .catch(err => {
+        throw new Error('Failed to fetch topics')
+      })
```

Comment:
This error may be considered redundant as you should be able to select from your topics